climada/util/plot.py

```diff
@@ -328,9 +328,21 @@ def geo_im_from_array(array_sub, coord, var_name, title,
     if not isinstance(axes, np.ndarray):
         axes_iter = np.array([[axes]])
 
-    if 'cmap' not in kwargs:
-        kwargs['cmap'] = CMAP_RASTER
-
+    # prepare colormap
+    cmap = plt.get_cmap(kwargs.pop("cmap", CMAP_RASTER))
+    cmap.set_bad("gainsboro")  # For NaNs and infs
+    cmap.set_under("white", alpha=0)  # For values below vmin
+
+    # prepare boundary values
+    if "norm" in kwargs:
+        min_value = kwargs["norm"].vmin
+        vmin = None  # We will pass norm
+        del kwargs["vmin"], kwargs["vmax"]
+    else:
+        min_value = np.nanmin(array_sub)
+        vmin = kwargs.pop("vmin", min_value)
+    below_value = min_value/2 if min_value > 0 else min_value-1
```

Comment:
`min_value` is not needed anymore afterwards, so we do not need to define a new variable
```suggestion
    min_value = min_value/2 if min_value > 0 else min_value-1
```